test/e2e/page-objects/pages/dialog/snap-install.ts

```diff
@@ -80,16 +85,8 @@ class SnapInstall {
     console.log(
       'Clicking on the scroll button and then clicking the confirm button',
     );
-    await this.driver.waitUntil(
-      async () => {
-        await this.driver.clickElementSafe(this.snapInstallScrollArea);
-        const isEnabled = await this.driver.findClickableElement(
-          this.nextPageButton,
-        );
-        return isEnabled;
-      },
-      { timeout: veryLargeDelayMs, interval: 100 },
-    );
+    await this.driver.waitForSelector(this.addToMetaMaskHeader);
+    await this.driver.clickElementSafe(this.snapInstallScrollArea);
```

Comment:
convenient that all tests use page objects, so one small fix could impact all specs, instead of having to go to individual specs, to see if they have flakiness at the same point